.github/workflows/ruby.yml

```diff
@@ -12,12 +12,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ '2.6', '2.7', '3.0' ]
+        ruby: [ '2.7', '3.0', '3.1', '3.2' ]
```

Comment:
ruby 2.7 is not supported anymore, so I guess we can drop it as well

Reply:
Agree. Can you disable the status check for the PR for 2.6 and 2.7 and add 3.1 and 3.2?

Reply:
looks like I do not have a proper role 🙂